src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -31,33 +31,44 @@ public static String buildSearchConceptsQuery(
     // domain clause filters for the given domain id based on field domain_id
     var domainClause = createDomainClause(conceptTablePointer, conceptTableVariable, domainId);
 
-    // search concept name clause filters for the search text based on field concept_name
-    var searchNameClause =
-        createSearchConceptClause(
-            conceptTablePointer, conceptTableVariable, searchText, "concept_name");
+    // if the search test is empty do not include the search clauses
+    // return all concepts in the specified domain
+    if (searchText == null || searchText.isEmpty()) {
+      // TODO: DC-845 Implement pagination, remove hardcoded limit
+      Query query =
+          new Query(List.of(nameField, idField), List.of(conceptTableVariable), domainClause, 100);
+      return query.renderSQL(platform);
+    } else {
```

Comment:
Using `else` after a `return` is unnecessary since the `return` has already interrupted the normal flow of execution.
```suggestion
      return query.renderSQL(platform);
    }
```
Either way works but, for me, omitting `else` makes it clearer.